packages/reporter/src/std/StdReporter.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { Reporter } from '../lib';
+import { Issue, Scan, Severity } from '../models';
+import { StdReporter } from './StdReporter';
+import { instance, mock, reset, when } from 'ts-mockito';
+
+const highSeverityIssue: Partial<Issue> = {
+  name: 'Reflective Cross-site scripting (rXSS)',
+  request: {
+    url: 'https://qa.brokencrystals.com/?artifical3160fc2b=%22%3Cdiv+OnCliCk%3Dalert%28576485%29%3E%3C%2Fdiv%3E'
+  },
+  severity: Severity.HIGH
+};
+
+const mediumSeverityIssue: Partial<Issue> = {
+  name: 'Directory Listing',
+  request: {
+    url: 'https://qa.brokencrystals.com/?'
+  },
+  severity: Severity.MEDIUM
+};
+
+const lowSeverityIssue: Partial<Issue> = {
+  name: 'Misconfigured X-Content-Type-Options Header',
+  request: {
+    url: 'https://qa.brokencrystals.com/'
+  },
+  severity: Severity.LOW
+};
+
+describe('StdReporter', () => {
+  let reporter!: Reporter;
+
+  const MockedScan = mock<Scan>();
+
+  beforeEach(() => {
+    /* eslint-disable no-console */
+    console.log = jest.fn();
+    console.warn = jest.fn();
+    console.error = jest.fn();
+    /* eslint-enable no-console */
+
+    reporter = new StdReporter();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+    reset<Scan | Console>(MockedScan);
+  });
+
+  describe('report', () => {
```

Comment:
Why don't add a couple of tests to verify the output structure?

```ts
verify(console.log(match('some string')).once();
```

In addition, I would mock the `table` function from the `tty-table` package to check that passed headers are generated properly.

Reply:
added some checks for table header and structure, but w/o `tty-table` mocks as imo it's mutable implementation detail